content/4_Gold/BFS.mdx

```diff
@@ -69,6 +69,32 @@ System.out.println(q.peek()); // 3
 
 </JavaSection>
 
+<PySection>
+
+#### Python
+
+Python has a `queue` built-in module.
```

Comment:
queue's synchronized, so it's a bit slow
perhaps just tell them to use deque instead?